cpp/src/cider/type/plan/BinaryExpr.cpp

```diff
@@ -36,6 +36,7 @@ void BinOper::initAutoVectorizeFlag() {
       case kPLUS:
       case kMINUS:
       case kMULTIPLY:
+      case kDIVIDE:
```

Comment:
```Div``` can't be vectorizable.

Reply:
enable Div for complex arithmetic operations, `i8*i16*2 + i16/3 - 1` gain 600%+ boost.
if we don't enable Div，`i8*i16*2 + i16/3 - 1` is slow than velox.

Reply:
Hm, I found div with literal is vectorizable, interesting.

Reply:
ok, I'll narrow auto-vectorization if the dividend is constant.